### DISCUSSION TITLE
next/Image component is not amp compliant

### DISCUSSION THREAD

**Discussion by olikami** [vercel/next.js#21189] [Ideas]
**Describe the feature you'd like to request**

The `next/Image` component should automatically create a valid `<amp-img>` tag with the responsive sizes if the page is being rendered as amp.

Currently, the amp validation (correctly) shows the following error: ` error  The parent tag of tag 'img' is 'div', but it can only be 'i-amphtml-sizer-intrinsic'.`

**Describe the solution you'd like**

I would like to be able to use the `next/Image` component directly for html and amp pages. I would just put `<Image>` and next would automatically use the current Image tag for normal pages and use an optimised (including responsive sizes) `<amp-img> tag. 

**Describe alternatives you've considered**

I considered using the `useAmp()` hook, but while it's possible I lose all the optimisations by the `next/Image` component.

**Comment by Siggnja** [comment]
Any news here?
I actually automatically expected this to work because of next.js tight integration with amp.

**Reply by SergeyRe** [reply]
this feature presented in Eleventy with[ AMP plugin ](https://blog.amp.dev/2020/07/28/introducing-the-eleventy-amp-plugin/)
For me its the reason to switch to 11ty for pure AMP projects

**Comment by DevinWallKcl** [comment]
I know this a super old discussion but did this just fall off the face of the earth? As far as Im aware this still isn't something that is supported correct?
